packages/frontend/app/(dashboard)/course/[cid]/queue/[qid]/components/modals/EditQueueModal.tsx

```diff
@@ -679,11 +680,25 @@ const EditQueueModal: React.FC<EditQueueModalProps> = ({
           </Form.List>
         </>
       )}
-      <Form.Item label="Zoom/Teams Link" name="zoomLink">
+      <Form.Item
+        label={
+          <div className="flex flex-row items-center gap-1">
+            Zoom/Teams Link
+            <Tooltip
+              title={
+                'This is the queue-specific link. If not set, the link shown to students will default to the course-wide link.'
+              }
+            >
+              <QuestionCircleOutlined style={{ color: 'gray' }} />
+            </Tooltip>
+          </div>
+        }
+        name="zoomLink"
+      >
         <Input
           allowClear={true}
           className="text-sky-800"
-          placeholder="[No Zoom/Teams link set]"
+          placeholder={course?.zoomLink ?? '[No Zoom/Teams link set]'}
```

Comment:
Huh interesting thing with this. So I guess nullish coalescing operator only checks if the value is null or undefined, not if it's falsy. I guess in this case all you would want to do is replace it with an `||`
```suggestion
          placeholder={course?.zoomLink || '[No Zoom/Teams link set]'}
```
![image](https://github.com/user-attachments/assets/a39f0516-2991-4dcd-b43a-675ab299447c)
![image](https://github.com/user-attachments/assets/8e2e0948-aa1e-4c23-9851-04776c29124c)

This made me realise I may have been using the nullish coalescing operator in places I shouldn't have been using it lol, and I should have been using || instead